integrations/create_no_session_test.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
```

Comment:
Tiny nitpick, but could you update year?

Reply:
```suggestion
// Copyright 2019 The Gitea Authors. All rights reserved.
```